go.mod

```diff
@@ -1,74 +1,48 @@
-module terraform-provider-prefect
+module terraform-provider-prefect2
```

Comment:
Go modules use the repository path (so our previous module name was incorrect), can you change this `github.com/PrefectHQ/terraform-provider-prefect` instead?

Reply:
hmm I'm not sure how we get this to work. When I change this to `github.com/PrefectHQ/terraform-provider-prefect` I get first an error where I reference the module, which is fine, but then when I update it I get this second error complaining about `import cycle not allowed`.

Do you have any thoughts @jawnsy ?

![image](https://user-images.githubusercontent.com/33239635/232652847-199fac44-9ec1-498c-8dab-9889602fa2ba.png)

Reply:
No worries! Leave this as it is, I'll take a look after we merge the initial version